nginx/static

```diff
@@ -0,0 +1,60 @@
+# You may add here your
+# server {
+#	...
+# }
+# statements for each of your virtual hosts to this file
+
+##
+# You should look at the following URL's in order to grasp a solid understanding
+# of Nginx configuration files in order to fully unleash the power of Nginx.
+# http://wiki.nginx.org/Pitfalls
+# http://wiki.nginx.org/QuickStart
+# http://wiki.nginx.org/Configuration
+#
+# Generally, you will want to move this file somewhere, and start with a clean
+# file but keep this around for reference. Or just disable in sites-enabled.
+#
+# Please see /usr/share/doc/nginx-doc/examples/ for more detailed examples.
+##
+
+server {
+        # listen 8080 default_server;
+	listen 80;
+	# listen [::]:80 default_server ipv6only=on;
+
+	root /srv/static;
+	# index index.php index.html;
+        # autoindex off;
```

Comment:
Does this turn off showing an index of all the files? I think we want that to be off, but what do you think?

Reply:
Yeah, I think we should turn it off too. The default is `off`. I will keep this in the config to keep it explicit.